src/BloomExe/Publish/PublishModel.cs

```diff
@@ -404,6 +408,8 @@ public IEnumerable<HtmlDom> GetPageDoms()
 					//get the original images, not compressed ones (just in case the thumbnails are, like, full-size & they want quality)
 					BookStorage.SetBaseForRelativePaths(previewXmlDocumentForPage, BookSelection.CurrentSelection.FolderPath);
 					HtmlDom.AddPublishClassToBody(previewXmlDocumentForPage.RawDom);
+					if(LayoutPagesForRightToLeft)
+						HtmlDom.AddRightToLeftClassToBody(previewXmlDocumentForPage.RawDom);
 					HtmlDom.AddHidePlaceHoldersClassToBody(previewXmlDocumentForPage.RawDom);
```

Comment:
We seem to be up to about 4 lines of code that are basically the same in three places; is there enough of a common theme for a method?